diff --git a/litgpt/transformers_local/llama/modeling_llama.py b/litgpt/transformers_local/llama/modeling_llama.py
index a929116..978f159 100644
--- a/litgpt/transformers_local/llama/modeling_llama.py
+++ b/litgpt/transformers_local/llama/modeling_llama.py
@@ -359,7 +359,7 @@ class LlamaModel(LlamaPreTrainedModel):
         # Initialize weights and apply final processing
         self.post_init()
 
-    @check_model_inputs()
+    @check_model_inputs
     @auto_docstring
     def forward(
         self,
diff --git a/litgpt/transformers_local/qwen3/modeling_qwen3.py b/litgpt/transformers_local/qwen3/modeling_qwen3.py
index c4253bd..45c3577 100644
--- a/litgpt/transformers_local/qwen3/modeling_qwen3.py
+++ b/litgpt/transformers_local/qwen3/modeling_qwen3.py
@@ -361,7 +361,7 @@ class Qwen3Model(Qwen3PreTrainedModel):
         # Initialize weights and apply final processing
         self.post_init()
 
-    @check_model_inputs()
+    @check_model_inputs
     @auto_docstring
     def forward(
         self,
